.github/workflows/recent_fail_summary.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Recent Fail Summary
+on:
+  schedule:
+    - cron: "*/10 * * * *"
```

Comment:
Do we want this to run this often?  How long does this workflow take to run?

In general, I would have thought that running this hourly or so would be fine...  In either case, some sort of reasoning behind the timing should be in a comment here.

Reply:
The workflow should take under a minute to run; every 10 minutes was just a placeholder but after reviewing how often pull requests are made I suppose it makes more sense to run this once daily for starters.